src/PIL/ImageMode.py

```diff
@@ -13,18 +13,21 @@
 # See the README file for information on usage and redistribution.
 #
 
+from PIL import Image
```

Comment:
This creates a circular import (not sure why it didn't crash CI). `PIL.Image` now uses `ImageMode.getmode` in `_conv_dtype_shape` and `PIL.ImageMode` uses `PIL.Image._ENDIAN`. As a result, both modules import each other.

Here is the import in the pillow module:

https://github.com/python-pillow/Pillow/blob/7b3f28047fe008d70bda1938e3178da388ecaa64/src/PIL/Image.py#L50

Do you think this will become problematic?

Reply:
Thanks for catching! I've pushed a commit to avoid the circular dependency.